CONTRIBUTION.md

```diff
@@ -0,0 +1,31 @@
+Contributing
+============
+
+It's good to hear that you want to contribute to Pony!
+
+First of all please [search existing issues][complete-issue-list]. Apart from bug reports you will
+find feature requests and work currently in progress too. If you cannot find a suitable issue —
+[create a new one][new-issue]. Potential pull requests are no exception. Before you start working,
+lets have a discussion and agree on the solution and scope.
+
+Each of [bug report](#bug-report), [feature request](#feature-request) or [pull request](#pull-request)
+has to cover different aspects described below.
+
+Bug report
+----------
+Provide a code example together with expected and actual results. If possible, try to isolate the
+problem and provide just enough code to demonstrate it. Add any related information which might help
+to fix the issue.
+

```

Comment:
We should explicitly mention including the OS used, it can make a huge difference when debugging and people so often miss it out.